config/system.php

```diff
@@ -70,6 +70,17 @@
 
     'charset' => 'UTF-8',
 
+    /*
+    |--------------------------------------------------------------------------
+    | CSV Export Delimiter
+    |--------------------------------------------------------------------------
+    |
+    | Statamic will use this character as delimiter for csv exports. 
+    |
+    */
+
+    'csv_delimiter' => ',',
+
```

Comment:
Can you please move this into `config/forms.php`?

Also if you remove the space after `exports. ` in the comment, StyleCI will stop complaining.